src/services/ExposureNotificationService/ExposureNotificationServiceProvider.tsx

```diff
@@ -60,6 +61,16 @@ export const ExposureNotificationServiceProvider = ({
     const onAppStateChange = async (newState: AppStateStatus) => {
       captureMessage(`ExposureNotificationServiceProvider onAppStateChange: ${newState}`);
       if (newState !== 'active') return;
+      if (Platform.OS === 'android' && Platform.Version >= 28) {
+        const notificationPayload: NotificationPayload = {
+          alertTitle: i18n.translate('Notification.ReminderTitle'),
+          alertBody: i18n.translate('Notification.ReminderBody'),
+          initialDelay: 60 * 24,
+          repeatInterval: 60 * 24,
+          flexTimeInterval: 60,
```

Comment:
I changed this to:
```
          initialDelay: 0,
          repeatInterval: 1,
          flexTimeInterval: 1,
```
and tested locally and I wasn't able to get any notifications. I put a console log inside the above if statement and it was executing.